app/models/respondent.rb

```diff
@@ -0,0 +1,72 @@
+class Respondent < ApplicationRecord
+	include SpotlightSearch
+	include Paginator
+
+	belongs_to :user
+	belongs_to :response_round
+	belongs_to :organisation
+	has_many :consultation_response
+
+	scope :search_user_query, lambda { |query|
+		return nil if query.blank?
+		terms = query.downcase.split(/\s+/)
+		terms = terms.map { |e|
+			(e.gsub("*", "%").prepend("%") + "%").gsub(/%+/, "%")
+		}
+		num_or_conds = 3
+		includes(:user).references(:user).
+		where(
+			terms.map { |term|
+				"(LOWER(users.first_name) LIKE ? OR LOWER(users.last_name) LIKE ? OR LOWER(users.email) LIKE ? )"
+			}.join(" AND "),
+			*terms.map { |e| [e] * num_or_conds }.flatten,
+		)
+	}
+
+	def self.invite_respondent(consultation, organisation, params)
```

Comment:
@BALAJI-BALA - there is a better way to define class methods and actually group them like so - 
```
  class << self
    def invite_respondent(consultation, organisation)
      #content goes here
    end
  end
```